Added "About the Support Team" page

williln opened the pull request.

phalt commented on general/support-team.md:
```diff
@@ -0,0 +1,101 @@
+
+
+
+
```
whitespace! :scream: :laughing:

williln commented:
Oh noes @phalt! I fixed it. Some of the lines you highlighted I had to leave because of paragraph breakage, but I removed everything I could! Feel free to submit a PR, too. :dolphin:

phalt commented:
@williln sorry Lacey I think I'm just being picky :laughing: it's totally fine.
